src/provider.ts

```diff
@@ -17,21 +18,21 @@ export interface IProviderOptions {
 	showHeader?: boolean;
 	leftArrow?: string;
 	rightArrow?: string;
-	
```

Comment:
Your IDE is removing spaces on "empty" lines. If the whole project has those spaces, we probably should keep those spaces.